custom_components/powercalc/data/ikea/LED1925G6/model.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Yes. I have 2 of this, and 2 of the other type (the white only one). So I redo the measurements with two lamps parallel. Do we need the same for the white only bulbs as well?

Reply:
That would be great. Just make sure to input the right amount of lights in the wizard. It will divide the measured power with the number of lights.

Reply:
Hm, wait. I just wanted to run the script in docker in two separate screens parallel. What wizard are you mentioning which can handle more than one lights? Sorry for the silly question, i did not find anything about this in the docs.

Reply:
Ok.

Reply:
Btw by parallel I meant you have to connect them elecronically in parallel (ie use a light fixture with 2 fittings) behind the Shelly smart plug.
You also need to control the Hue group in this case so both lights are set to the same color / brightness.
There is a question in the measure tool: "Are you measuring multiple lights. In some situations it helps to connect multiple lights to be able to measure low currents.". Just select yes here and it will ask for the number of connected light bulbs.

Reply:
Just a simple question. 
How can you enter 2 different lighs in the script. I am not able to select multiple ones, just only one.
? Are you measuring multiple lights. In some situations it helps to connect multiple lights to be able to measure low currents.  Yes
? How many lights are you measuring?  2
? Select the light?  light.color_bulb_1
? What model is your light? Ex: LED1837R5  LED1925G6
2022-03-12 14:50:31,236 [INFO] Measuring standby power. Waiting for 20 seconds...

Reply:
They are indeed plugged into the same socket. Not is the issue. But how can i control them parallel, as the script does let me select only one light.

Reply:
Ah. OK. I think, got it. Need to set up a hue group in HA :)

Reply:
Sorry for the silly question, i tought, the script itself will manage these lights parallel not over an HA group :) so i misslead myself. :)

Reply:
There are no dumb questions ;-). Yes you need to setup a group in HA or alternatively Hue group (but that will only work when you use the hue bridge).

Reply:
Well, Seeing the values which varies a lot even in nearly standby, when the bulb is not really emitting light i will redo the measurement with the white bulbs as well, as i have two of them also. Better to have it right, not just guessed :) so when this is done, I will commit the files.

Reply:
Thanks for your efforts. And have a nice weekend

Reply:
I uploaded the new files.